docs/ref/release_notes.rst

```diff
@@ -1,6 +1,42 @@
 Release Notes
 =============
 
+
+What's new in iris-grib v0.15
+-----------------------------
+
+:Release: 0.15.0
+:Date: 4 Dec 2019
+
+Features
+^^^^^^^^
+
+* Updated translations between GRIB parameter code and CF standard_name or
+  long_name
+
+* Added support for loading GRIB messages with no fixed surface set in the
+  product definition section
+
+* Added support for loading GRIB messages where i or j increment are not set
+
+* Added support for saving cubes that have a "depth" coordinate
+
+* Cubes loaded from GRIB files now contain a new GRIB_PARAM attribute, the
+  value of which is an instance of
+  iris_grib.grib_phenom_translation.GRIBCode and represents the parameter code
```

Comment:
```suggestion
  iris_grib.grib_phenom_translation.GRIBCode and represents the parameter code.
  When saving, if a cube has a GRIBCode attribute, this determines the parameter coding
  of the created message(s): This will _override_ any translation from the CF names.
```